tests/compile-all/build.sh

```diff
@@ -120,7 +120,7 @@ do
             example_dir=`dirname "$example"`
 
             # Build the goal
-            $LOG_INFO make -C "$example_dir" -j TARGET=$platform BOARD=$board $GOAL
+            $LOG_INFO make -C "$example_dir" -j TARGET=$platform BOARD=$board clean $GOAL
```

Comment:
This one just adds the "clean" to the log. I'm assuming that you also meant to add it to the compile command - will do this change myself.